src/chapters/data-and-variables/input.rst

```diff
@@ -125,8 +120,7 @@ prints their sum:
 
 Do you see the output you expected?
 
-If we enter ``7`` and ``2``, we expect an output of ``9``.  We do NOT expect
-``72``, but that is the result printed. What gives?!?!?
+If we enter ``7`` and ``2``, we may expect an output of ``9``. The result printed is 72. What gives?!?!?
```

Comment:
as you've pointed out to me elsewhere, it's not a great strategy to assume what the reader is thinking. and we, the textbook authors, don't expect that output

Reply:
Yesssss... but I think that with the numbers 7 + 2, we can safely assume that the students will expect a result of 9.  The point of the repl and discussion is to demonstrate a common mistake when collecting user input.

How about changing the wording to avoid reading minds?  Something like, "Adding the numbers ``7`` and ``2`` should give..."